### DISCUSSION TITLE
No Scrobble to Last.fm button?

### DISCUSSION THREAD

**Discussion by worf1337** [navidrome/navidrome#3056] [Q&A] [answered]
Hey all, just updated Navidrome to latest via Docker (0.52.5). Since then the switch to enable Lastfm scrobbling is gone. Is this expected?

Thanks.

**Comment by deluan** [comment, marked as answer]
If you never configured your own Last.fm API Key, then yes, this changed in [v0.51.0](https://github.com/navidrome/navidrome/releases/tag/v0.51.0). See details and instructions in the release notes.

If you had created your own API key before, then it may be a bug. What was your prior version before the upgrade?

Also, there was a small bug in [0.51.1](https://github.com/navidrome/navidrome/releases/tag/v0.51.1) that also broke Last.fm integration. If that is your case you can simply logoff, refresh the browser and login again.

**Comment by worf1337** [comment]
ahhhh ok, got it working thanks so much
